bot/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM python:3-alpine
 
-COPY bot /src
-RUN pip install /src
+RUN apk add --update gcc musl-dev libffi-dev openssl-dev
```

Comment:
This will make the image quite bigger, you could do:
```
ADD tools /src/tools
ADD bot /src/bot

RUN apk add --update ... && cd /src/tools && python setup.py install && ... && cleanup apk
```

Reply:
[Before](https://tools.taskcluster.net/groups/BR2ToA1oTO2g3uT74hD_8Q/tasks/BtMg_AiDRHuT1ZbhsgQhcw/runs/0/artifacts): 87,37 Mb
[Now](https://tools.taskcluster.net/groups/Cgj4BMKpSf2qogC7lb1-0w/tasks/eGgJ8uxuSeqRdZMiHhnMDQ/runs/0/artifacts): 41,49 M :tada: